docs/ref/models/instances.txt

```diff
@@ -528,8 +528,8 @@ Updating attributes based on existing fields
 --------------------------------------------
 
 Sometimes you'll need to perform a simple arithmetic task on a field, such
-as incrementing or decrementing the current value. The obvious way to
-achieve this is to do something like::
+as incrementing or decrementing the current value. You can achieve this by
+doing something like::
```

Comment:
```suggestion
as incrementing or decrementing the current value. One way of achieving this
is doing the arithmetic in Python like::
```

I think this captures that it's "a way" of doing it, and signposts that the following paragraphs will explain a "better" way.

I'm not sure about "common" - maybe it's common, maybe it's not, and that doesn't say whether or not it's going to be problematic.

Reply:
Applied.